form-flow/src/main/resources/config/form-flow/schema/formflow.schema.json

```diff
@@ -75,10 +75,6 @@
                                 "step"
                             ]
                         }
-                    },
-                    "nextStep:": {
-                        "type": "string",
-                        "description": "The key of the next step in the flow. When multiple steps should be available you should use the property 'nextSteps'"
                     }
                 },
                 "required": [
```

Comment:
You can now make nextSteps required right?

Reply:
ah, yes. Will do

Reply:
I did say yes at first, but a final step will not have next steps.

Reply:
ah yeah of course